src/pages/index.js

```diff
@@ -36,9 +38,34 @@ const initialCards = [
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
   },
 ];
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "26a66bc9-8fc9-40b5-980e-7373cd063117",
+    "Content-Type": "application/json",
+  },
+});
+
+// Destructure the second item in the callback of the .then()
+api
+  .getAppInfo()
+  .then(([cards]) => {
```

Comment:
* When I reload the page, the old `avatar` and `profile` data appears again

You get `userData` as the 2nd argument in the array. Please, use it for setting the profile data and `avatar` here


```js
api
  .getAppInfo()
  .then(([cards, userData]) => {  // userData comes here
    cards.forEach((item) => {
      const cardEl = getCardElement(item);
      cardsList.append(cardEl);
    });

    //Handle the user's information
    // - set the src of the avatar image
    // - set the textcontent of both the text elements
  })
  .catch(console.error);

```